ui/app/adapters/pki/pki-role-engine.js

```diff
@@ -0,0 +1,3 @@
+import PkiRoleAdapter from './pki-role';
```

Comment:
extending the current PKI adapter. This allows us to take advantage of the current network requests that we will need, while also allowing us to add more network requests in this file that are not used in the current PKI engine. When we complete this work, we can think about refactoring the current PKI adapter, but we determined that was outside the scope of this initial phase (just getting the work done and functional). The PKI adapter is dependent on other adapters and kind of a maze to work through. It's not an easy task to refactor at this point.